### DISCUSSION TITLE
The Future Ibax

### DISCUSSION THREAD

**Discussion by Cakhoo** [IBAX-io/go-ibax#1584] [Soru Cevap - Türkçe]
Ladies and gentlemen / hello .
What do you think about the future of the Ibax coin?
Are we going to great places? or will it be an unlucky project that we did not expect?
Let me get the predictions...

**Comment by Yusufknk** [comment]
İbax kendi ekosistemine sahip nadir projelerden biri. İleride güzel yerlere gelecektir. Bunun için zaman lazım
